### DISCUSSION TITLE
TabbyML vs Twinny

### DISCUSSION THREAD

**Discussion by dawidmachon** [rjmacarthy/twinny#410] [Q&A]
Hi,
Anyone could provide me what is a main difference between these two? Is worth switching from TabbyML to Twinny?

**Comment by rjmacarthy** [comment]
Hey, thanks for the question.  I guess TabbyML has a lot more financial backing and support. This is a personal project which has some community backing and may not be as feature rich etc than others.  It's also 100% free and doesn't require signup or anything like that it's configurable by default and should work with a number of different providers if configured correctly e.g ollama, lm studio, llamacpp etc.  Many thanks!
